app/assets/stylesheets/components/common/_newsletter_signup.scss

```diff
@@ -6,6 +6,11 @@
   @include body(16, 22);
 }
 
+.t-newsletter-strap {
```

Comment:
No, this isn't for testing - the t- class was already in use so I just kept it. I'll change that where it appears.